src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateValue.txt

```diff
@@ -1,6 +1,12 @@
 ﻿
 // Use of DateValue, DateTimeValue
 
+>> DateValue("")
```

Comment:
Could you add a case for Blank() as a parameter?

Reply:
Good idea, added as well